src/extent.c

```diff
@@ -2279,3 +2279,23 @@ extent_boot(void) {
 
 	return false;
 }
+
+int
+extent_frag_stats(tsdn_t *tsdn, const void *ptr,
+    size_t *nfree, size_t *nregs, size_t *size) {
+	const extent_t *extent = iealloc(tsdn, ptr);
+	if (unlikely(extent == NULL)) {
+		*nfree = *nregs = *size = 0U;
+		return EFAULT;
```

Comment:
nit: this can simply return true; the EFAULT etc is for user-facing return codes. Internal functions don't really need to use them.